### PR TITLE
feat: add password confirm box at signup

### PR DESCRIPTION
**Pull request by ajskateboarder**
This adds a "Confirm password" box in the join screen. As you might know, this is helpful for users who might mistype their password when making their account.

meower

### CODE REVIEW

**Review by tnix100 — APPROVED** [review]
LGTM, nice feature to have

File: src/screens/Setup.svelte [hunk 1]
```diff
@@ -331,11 +331,18 @@
...
+					const confirmPassword = e.target[2].value;
+
 					if (!(username && password)) {
 						loginStatus =
 							"You must specify a username and a password to create an account!";
 						return false;
 					}
+					if (!(password === confirmPassword)) {
+						loginStatus =
+							"Make sure you have entered your password correctly!";
```
[the remark this answers is not in the document]
Comment (lemystic) [reply src/screens/Setup.svelte]: This is much friendlier 👍

**Review by CST1229 — APPROVED** [review]

**Comment by ajskateboarder** [comment]
What happened to this PR @tnix100 @CST1229

**Comment by tnix100** [comment]
> What happened to this PR @tnix100 @CST1229

Will merge soon.

**Comment by tnix100** [comment]
Closing as this has been re-implemented on the develop branch. I didn't merge because of too many merge conflicts, but I added you as a co-author on my commit so you were still credited.

**Pull request closed without merging**